### PR TITLE
Tanzanian Consulate Bill

### PR DESCRIPTION
**Pull request by godlyeriss**

### CODE REVIEW

File: Tanzanian Consulate Act, adding two additional ministerial foreign (to tanzania) positions.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    may also do something called a Cogo te ad suffragium mecum ten times, or, 
+    "I force you to vote with me," shortened to Ctasm, which forces the
+    Grand Consul's vote (in to agree with the Supreme Consul's. The
+    Supreme Consul is to have one vote in the Ministry.
+</p>
+<h4>Grand Consul</h4>
+<p>
+    The Grand Consul is to have one vote in the Ministry and the ability to
+    transfer voting power in the ministry to the Supreme Consul for up to
+    one week at a time up to five times.
```
Comment (TheSuperGamer20578) [inline Tanzanian Consulate Act, adding two additional ministerial foreign (to tanzania) positions.html:25]: Being able to stop the Ministry from voting is probably a bad idea.

**Review by TheSuperGamer20578 — CHANGES_REQUESTED** [review]
Does not state how many people run for each position.

File: Tanzanian Consulate Act, adding two additional ministerial foreign (to tanzania) positions.html (hunk 1 above)
Comment (godlyeriss) [reply Tanzanian Consulate Act, adding two additional ministerial foreign (to tanzania) positions.html:25]: one each. it doesnt stop the ministry it just transfers their voting power to the supreme consul

Comment (TheSuperGamer20578) [reply Tanzanian Consulate Act, adding two additional ministerial foreign (to tanzania) positions.html:25]: If it transfers voting power, the Ministry won't be able to vote.

Comment (godlyeriss) [reply Tanzanian Consulate Act, adding two additional ministerial foreign (to tanzania) positions.html:25]: no it gives the Grand Consul's vote to the Supreme Consul

Comment (TheSuperGamer20578) [reply Tanzanian Consulate Act, adding two additional ministerial foreign (to tanzania) positions.html:25]: It clearly says "transfer voting power in the ministry to the Supreme Consul", it implies the whole ministry.

Comment (godlyeriss) [reply Tanzanian Consulate Act, adding two additional ministerial foreign (to tanzania) positions.html]: oh i meant transfer the grand consul's power

File: Tanzanian Consulate Act; Adding two additional ministerial foreign (to tanzania) positions.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    may also do something called a Cogo te ad suffragium mecum ten times, or, 
+    "I force you to vote with me," shortened to Ctasm, which forces the
+    Grand Consul's vote (in to agree with the Supreme Consul's. The
+    Supreme Consul is to have one vote in the Ministry.
+</p>
+<h4>Grand Consul</h4>
+<p>
+    The Grand Consul is to have one vote in the Ministry and the ability to
+    transfer voting power in the ministry to the Supreme Consul for up to
+    one week at a time up to five times.
```
Comment (TheSuperGamer20578) [inline Tanzanian Consulate Act; Adding two additional ministerial foreign (to tanzania) positions.html:25]: If a term is one month and they can do this for up to a week at a time, wouldn't 5 times mean they can do it for, knger than their term? Or is ommiting per term done on purpose and the position can only ever do it 5 times? 
```suggestion
    one week at a time up to three times each term.
```

File: Tanzanian Consulate Act; Adding two additional ministerial foreign (to tanzania) positions.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+<h4>Elections</h4>
+<p>
+    The Grand and Supreme Consuls are to be elected by majority by the electorate
+    within Sudan.
+</p>
+<h3>Term Length</h3>
+<p>
+    The Grand and Supreme Consuls' term lengths are to be the same as any other minister,
+    one month.
+</p>
```
Comment (TheSuperGamer20578) [inline Tanzanian Consulate Act; Adding two additional ministerial foreign (to tanzania) positions.html:36]: This does not state when the elections should happen or how long the campeigning period should be.

**Review by TheSuperGamer20578 — CHANGES_REQUESTED** [review]

File: Tanzanian Consulate Act; Adding two additional ministerial foreign (to tanzania) positions.html (hunk 2 above)
Comment (godlyeriss) [reply Tanzanian Consulate Act; Adding two additional ministerial foreign (to tanzania) positions.html:25]: yeah do that

File: Tanzanian Consulate Act; Adding two additional ministerial foreign (to tanzania) positions.html (hunk 3 above)
Comment (TheSuperGamer20578) [reply Tanzanian Consulate Act; Adding two additional ministerial foreign (to tanzania) positions.html:40]: Read Section [2.1](http://emc-sudan.herokuapp.com/laws/Constitution#2.1) of the Constitution for an example.

**Review by TheSuperGamer20578 — APPROVED** [review]

**Pull request merged**